jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1,8 +1,8 @@
 ;; -*- mode: lisp;-*-
 
-(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/include/baxter-interface.l")
-(require "package://jsk_2015_05_baxter_apc/euslisp/jsk_2015_05_baxter_apc/util.l")
-(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l")
+(require :baxter "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l")
+(require :baxter-interface "package://baxtereus/baxter-interface.l")
+(require :util "package://jsk_2015_05_baxter_apc/euslisp/jsk_2015_05_baxter_apc/util.l")
```

Comment:
provideを書くと、ロードされる側でモジュール名を決められるということなのでは？

--
和田 健太郎 / Kentaro Wada
http://wkentaro.com/

2016年4月29日 22:01 +0900 に Shingo Kitagawa<notifications@github.com>は書きました:
>  
> Injsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l(https://github.com/start-jsk/jsk_apc/pull/1365#discussion_r61572801):
>  
> >@@ -1,8 +1,8 @@>;; -*- mode: lisp;-*->>-(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/include/baxter-interface.l")>-(require "package://jsk_2015_05_baxter_apc/euslisp/jsk_2015_05_baxter_apc/util.l")>-(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l")>+(require :baxter "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter.l")>+(require :baxter-interface "package://baxtereus/baxter-interface.l")>+(require :util "package://jsk_2015_05_baxter_apc/euslisp/jsk_2015_05_baxter_apc/util.l")
>  
> loadする方にprovideを書く必要はないんですか？
>  
>  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly orview it on GitHub(https://github.com/start-jsk/jsk_apc/pull/1365/files/ede0a55ffc740ea878e7b6b2d718529dfb8527ec#r61572801)
>  
>  
>